libs/accounts/src/lib/breakdown-table.tsx

```diff
@@ -0,0 +1,85 @@
+import type { CSSProperties } from 'react';
+import { forwardRef } from 'react';
+import type { ValueFormatterParams } from 'ag-grid-community';
+import { PriceCell, t } from '@vegaprotocol/react-helpers';
+import {
+  AgGridDynamic as AgGrid,
+  progressBarCellRendererSelector,
+} from '@vegaprotocol/ui-toolkit';
+import { AgGridColumn } from 'ag-grid-react';
+import type { AgGridReact, AgGridReactProps } from 'ag-grid-react';
+import type { AccountFields } from './accounts-data-provider';
+import { AccountTypeMapping } from '@vegaprotocol/types';
+import type { AccountType } from '@vegaprotocol/types';
+import {
+  assetDecimalsFormatter,
+  progressBarHeaderComponentParams,
+  progressBarValueFormatter,
+} from './accounts-table';
+
+interface BreakdownTableProps extends AgGridReactProps {
+  data: AccountFields[] | null;
+  style?: CSSProperties;
```

Comment:
chore (non blocking): Looks like this prop isnt used

Reply:
removed